.github/workflows/scratch-org-sfdx-ci-pr.yml

```diff
@@ -72,6 +72,12 @@ jobs:
         runs-on: ubuntu-latest
         needs: formatting-and-linting
         steps:
+            # Setup Java SDK for Salesforce CLI Scanner
```

Comment:
@msrivastav13 Is that really needed?
Looking at the spec of the [GitHub image](https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-README.md#java), we should already have a 1.8 JDK.

Reply:
@pozil Ah completely forgot that image has this and no wonder it took only 2 sec to install a JDK as it was just reading from cache!